src/navigation/screenOptions.tsx

```diff
@@ -75,8 +74,6 @@ export const getScreenOptions =
     };
   };
 
-/* eslint-enable complexity */
```

Comment:
If we now have standard tabbar sizes. How about doing the same for the header? Currently we are also using a customized sizing for the header.
If we want a standard header, the following codes can be deleted.

`src/navigation/screenOptions`

```
 const headerHeight = !isPortrait ? normalize(35) + insets.top : normalize(56) + insets.top;
...
headerStyle: {
  height: headerHeight
},
```

Reply:
Portrait
|before|after|
|--|--|
<img width="483" alt="image" src="https://github.com/user-attachments/assets/e4cdb6f3-08ab-4216-9910-606fdac2ba36" />|<img width="476" alt="image" src="https://github.com/user-attachments/assets/fbb4099a-15c5-4afe-aab0-55522641d218" />

Landscape
|before|after|
|--|--|
<img width="985" alt="image" src="https://github.com/user-attachments/assets/b9982c8c-3644-452e-b2a0-8a96737f49a7" />|<img width="978" alt="image" src="https://github.com/user-attachments/assets/ace7117e-8920-460f-adb4-391c531b29cd" />

Reply:
yeah I am completely convinced by the idea